### PR TITLE
move `clearborder` from Images.jl to ImageMorphology.j

### PR DESCRIPTION
**Pull request by mgautam98**
Solves #27 

Moved clearborder from Images.jl to ImageMorphology.jl with tests

### CODE REVIEW

**Comment by mgautam98** [comment]
@johnnychen94 please review it

File: src/clearborder.jl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+cleared_img = clearborder(img, width)
+cleared_img = clearborder(img, width, background)
+```
+Returns a copy of the original image after clearing objects connected to the border of the image.
+Parameters:
+ -  img          = Binary/Grayscale input image
+ -  width        = Width of the border examined (Default value is 1)
+ -  background   = Value to be given to pixels/elements that are cleared (Default value is 0)
+"""
+function clearborder(img::AbstractArray, width::Int=1, background::Int=0)
```
Comment (johnnychen94) [inline src/clearborder.jl:13]: Strictly speaking, this doesn't belong to this PR, but it would be great to have a patch here.

Generally speaking, annotating the type as `Int` is way too strict. we should use `Integer` instead.

```julia
julia> function foo(x::Int)
       @show x
       end
foo (generic function with 1 method)

julia> foo(UInt8(1))
ERROR: MethodError: no method matching foo(::UInt8)
Closest candidates are:
  foo(::Int64) at REPL[17]:2
Stacktrace:
 [1] top-level scope at REPL[22]:1
```

And **probably** would be convenient to also support `width::Real` when `isapprox(width % 1, 0)`.

**Comment by mgautam98** [comment]
I have attributed the original author from [b183cea3f8562986031dd0d8fdd52fb94708b16e](https://github.com/JuliaImages/Images.jl/commit/b183cea3f8562986031dd0d8fdd52fb94708b16e) and made the suggested changes

**Comment by johnnychen94** [comment]
Nice. Will be good to merge once the corresponding PR in Images.jl is submitted

**Comment by mgautam98** [comment]
Corresponding PR in Images.jl https://github.com/JuliaImages/Images.jl/pull/902
